local-volume/README.md

```diff
@@ -98,30 +98,81 @@ $ ALLOW_PRIVILEGED=true LOG_LEVEL=5 FEATURE_GATES=PersistentLocalVolumes=true ha
 
 ### Step 2: Creating local persistent volumes
 
-#### Option 1: Bootstrapping the external static provisioner
+#### Option 1: Generating Provisioner's daemonset spec and config map by using Helm chart 
```

Comment:
Option 1: Deploying the external static local provisioner.

Reply:
We should have 3 subsections:
1. Setting up admin account and cluster rolebindings for the provisioner
2. Generating and deploying the provisioner daemonset and configmap specs
3. After provisioner deployment